django/thunderstore/repository/admin/package_version.py

```diff
@@ -41,11 +42,19 @@ class PackageVersionAdmin(admin.ModelAdmin):
         "package__owner__name",
         "package__namespace__name",
         "version_number",
+        "file_tree__entries__blob__checksum_sha256",
     )
     date_hierarchy = "date_created"
-    readonly_fields = [x.name for x in PackageVersion._meta.fields] + [
+    readonly_fields = (
         "file_tree_link",
-    ]
+        "listings",
+    )
+    exclude = (
+        "website_url",
+        "file_tree",
+        "readme",
+        "changelog",
+    )
```

Comment:
Why are we excluding these? Now they're not accessible anywhere anymore via the admin